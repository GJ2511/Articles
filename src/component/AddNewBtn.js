import React from 'react';
import { Link } from 'react-router-dom';

function AddNewBtn() {
    return (
        <Link to="/new/article" className="btn btn-primary rounded-pill mb-2">
            + Add New Article
        </Link>
    );
}

export default AddNewBtn;
