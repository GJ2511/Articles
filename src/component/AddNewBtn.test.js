import React from 'react';

import AddNewBtn from './AddNewBtn';

describe('<AddNewBtn tests />', () => {
    test('match snapshot', () => {
        expect(shallow(<AddNewBtn />)).toMatchSnapshot();
    });
});
