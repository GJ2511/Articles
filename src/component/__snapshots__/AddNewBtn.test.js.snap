// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AddNewBtn tests /> match snapshot 1`] = `
<Link
  className="btn btn-primary rounded-pill mb-2"
  to="/new/article"
>
  + Add New Article
</Link>
`;
